cmd/podman/validate/args.go

```diff
@@ -47,6 +48,9 @@ func IDOrLatestArgs(cmd *cobra.Command, args []string) error {
 // CheckAllLatestAndCIDFile checks that --all and --latest are used correctly.
 // If cidfile is set, also check for the --cidfile flag.
 func CheckAllLatestAndCIDFile(c *cobra.Command, args []string, ignoreArgLen bool, cidfile bool) error {
+	if registry.IsRemote() {
+		return nil
```

Comment:
I still think this check is still needed. We are also looking at `args`

Reply:
ok, lemme see.